src/lit-html.ts

```diff
@@ -268,24 +268,41 @@ export class NodePart extends Part {
   setValue(value: any): void {
     value = this._getValue(value);
 
-    if (value instanceof Node) {
-      this._previousValue = this._setNode(value);
+    if (value === null ||
+        !(typeof value === 'object' || typeof value === 'function')) {
```

Comment:
Only because functions can have properties. I know boxed primitives can, but it seems very rare.